zds/member/models.py

```diff
@@ -360,6 +362,12 @@ def get_followed_topics(self):
         return Topic.objects.filter(topicfollowed__user=self.user)\
             .order_by('-last_message__pubdate')
 
+    def is_dev(self):
+        """
+        Check if the user is in a group contained in settings.ZDS_APP['member']['dev_group'].
```

Comment:
"contained in" sous-entend que `settings.ZDS_APP['member']['dev_group']` est une liste.

    Check whether user is part of group `settings.ZDS_APP['member']['dev_group']`.